### PR TITLE
Brexit blog post for GE2017

### PR DESCRIPTION
**Pull request by philipjohn**
This will be accessible via havesomenew.io/brexit which goes on our leaflets.

### CODE REVIEW

File: _posts/2017-05-15-ge2017-brexit.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+When asked to make our choice in the referendum, there was no plan presented for what a Leave vote would mean. Nor did the Remain campaign adequately address what it would do to address the EU's many failings. Had there been a guarantee of a referendum on the terms of Brexit in the event of a Leave vote or the reforms we would pursue under Remain, it would have given the electorate a much clearer indication of *what* they were voting for.
+
+Instead we now have politicians clambering to pursue their own agenda and claiming they have "the will of the people" or a "mandate" without having explicit public backing for their plans.
+
+## 37%
+
+Much is said about the 48%, but what about the 28% - the nearly 13m people who didn't vote? We cannot assume they were either Leave or Remain, and we should not ignore them.
+
+In fact, only 37% of the electorate (46,500,001) voted Leave. Such massive constitutional change should not be decided by a minority. We'd like to see referendums on serious constitutional change, such as Brexit, requiring 50% or more of the electorate (23,250,000 votes) to change the status quo.
```
Comment (Floppy) [inline _posts/2017-05-15-ge2017-brexit.md]: That 45 million figure is presumably the entire electorate? It reads like it's the 37% figure, to me...

File: _posts/2017-05-15-ge2017-brexit.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+## The Something New Solution
+
+We're clear on what we would do. If Brexit does go ahead we would;
+
+* Remain part of the European Economic Area (EEA), protecting the rights of Brits to move freely around Europe, and the rights of EU citizens to come to the UK, as so many do to live and work and better our society.
+* Remain part of the European Free Trade Association (EFTA), ensuring British businesses can trade with our biggest market free from the burden of punitive trade tarrifs.
+* We'll retain the benefit of the British laws that have resulted from our partnership with the EU, protecting the rights of British workers and ensuring the human rights of everyone.
+
+If Brexit doesn't happen, which is a possibility given the offensive negotiating stance of the Conservative government, we would;
```
Comment (Floppy) [inline _posts/2017-05-15-ge2017-brexit.md]: maybe add "and is still entirely possible, according to the EU"

File: _posts/2017-05-15-ge2017-brexit.md (hunk 1 above)
Comment (philipjohn) [inline _posts/2017-05-15-ge2017-brexit.md]: Yes, you're right, well spotted.

File: _posts/2017-05-15-ge2017-brexit.md (hunk 2 above)
Comment (philipjohn) [inline _posts/2017-05-15-ge2017-brexit.md]: Good idea!

File: _posts/2017-05-15-ge2017-brexit.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## The Something New Solution
+
+We're clear on what we would do. If Brexit does go ahead we would;
+
+* Remain part of the European Economic Area (EEA), protecting the rights of Brits to move freely around Europe, and the rights of EU citizens to come to the UK, as so many do to live and work and better our society.
+* Remain part of the European Free Trade Association (EFTA), ensuring British businesses can trade with our biggest market free from the burden of punitive trade tarrifs.
+* We'll retain the benefit of the British laws that have resulted from our partnership with the EU, protecting the rights of British workers and ensuring the human rights of everyone.
+
+If Brexit doesn't happen, which is a possibility given the offensive negotiating stance of the Conservative government, and is still entirely possible, according to the EU, we would;
```
Comment (philipjohn) [inline _posts/2017-05-15-ge2017-brexit.md]: Is this sentence too much now do you think, @Floppy ? There might be a better way to structure it, but I'm running out of brain power.

Comment (Floppy) [inline _posts/2017-05-15-ge2017-brexit.md]: Yeah, it's a bit run-on now. Probably just "If Brexit doesn't go ahead, which is still possible, we would:"

**Pull request merged**